packages/uswds-core/src/styles/mixins/general/table.scss

```diff
@@ -5,13 +5,12 @@
 @use "../utilities" as *;
 @use "../typography/typeset" as *;
 
-$table-text-color: color(
-  get-color-token-from-bg(
-    $bg-color: $theme-body-background-color,
-    $preferred-text-token: $theme-text-reverse-color,
-    $fallback-text-token: $theme-table-text-color,
-    $context: "Table text",
-  )
+
+$table-text-color:get-color-token-from-bg(
+  $bg-color: $theme-body-background-color,
+  $preferred-text-token: $theme-text-reverse-color,
+  $fallback-text-token: $theme-table-text-color,
```

Comment:
Shouldn't the default `$theme-table-text-color` be the preferred token?

```scss
$bg-color: $theme-body-background-color // "white"
$preferred-text-token: $theme-text-reverse-color, // "white"
$fallback-text-token: $theme-table-text-color, // default
```

Reply:
I agree. I will test out using the settings as the preferred token, not the fallback, throughout `table.scss`